.idea/.name

```diff
@@ -0,0 +1 @@
+website
```

Comment:
@shweta97 Remember to only include files in your commit which are logically related to the change you made. This file, for example, does not belong and should be removed.